pom.xml

```diff
@@ -96,7 +97,7 @@
         <dependency>
             <groupId>org.jboss.marshalling</groupId>
             <artifactId>jboss-marshalling-river</artifactId>
-            <version>1.4.12.jenkins-3</version> <!-- https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3 -->
+            <version>2.0.5.Final</version> <!-- https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3 -->
```

Comment:
Uh, if you are switching to a public release, please delete the XML comment explaining what was in the private release!

Reply:
CC @oleg-nenashev @svanoort